src/app/services/local-storage.service.ts

```diff
@@ -16,10 +17,17 @@ export class LocalStorageService extends StorageService {
    * @constructor
    * @param db
    */
-  constructor(private db: DexieStorage) {
+  constructor(
+    private db: DexieStorage,
+    private log: Logger
+  ) {
     super();
   }
 
+  async open(): Promise<void> {
+    return this.db.open().then(d => { return; });
```

Comment:
Es macht keinen großen Unterschied, aber ich fände es besser Promise.resolve() zu verwenden.

`return this.db.open().then(d => Promise.resolve());`